skylog/storage/cloudspanner/schema.sdl

```diff
@@ -0,0 +1,9 @@
+-- TreeNodes stores Merkle tree nodes. The nodes are sharded according to
+-- ShardId, which is computed based on the tree shape and write pattern, in
+-- order to maximize writing parallelism.
+CREATE TABLE TreeNodes (
+  TreeId       INT64 NOT NULL,
+  ShardId      INT32 NOT NULL,
+  NodeId       UINT64 NOT NULL,
```

Comment:
Possibly another key hotspot, though probably not as bad given it's a packed encoding.

Reply:
The idea is that if the `TreeOpts` is good then the load on a single shard will be `O(2^ShardLevels)` nodes/s, so we have control on this hotspot.

Reply:
SGTM.